CONTRIBUTING.md

```diff
@@ -70,12 +70,8 @@ first.
 
 First of all, navigate to the `mocks/oauth` directory.
 
-You can handle the SSL-support in two different ways:
-
-### Option 1: Use insecure connections
-
-This is the default and recommended option for local development.
-Do not use it in production!
+This is the recommended option for local development.
+Do not use insecure connections it in production!
```

Comment:
```suggestion
```

I'd remove the two lines. Not only insecure connections, but the OAuth mock in general should not be used in production.

"This is the recommended option for local development." doesn't make much sense anymore as we have only one option left.

Instead, I would add `Do not use the OAuth mock in production!` after `For local development, we have an OAuth mock that needs to be started first.`